test/linalg1.jl

```diff
@@ -2,7 +2,7 @@ debug = false
 
 import Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
-n = 10
+n = 10  # WARNING: n must be >= 10 for GeneralizedSchur tests
```

Comment:
No, just in all of the tests that relate to the Generalized Schur method (both mine and the original generalized Schur tests), they get two matrices by splitting `a` into 2 matrices -- `a[1:5, 1:5], a[6:10, 6:10]`.  Just thought it was worth mentioning in case some one ever changed it (the alternative could be to index based on `n`).